### ISSUE TITLE
"Segmentation fault (core dumped)" after update

### ISSUE THREAD

**Issue by aris-t** [ultralytics/yolov5#7424]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### YOLOv5 Component

Other

### Bug

I can't even get anything to run. After running git pull I cannot even get anything to run. It immediately errors out with Segmentation fault (core dumped). No other error information is given. This happens on all of the default example commands.

What I have tried:
* Fresh git clone.
* Pip install yolov5
* Yes I did install the requirements, I had them all already.

What I suspect:
* Pytorch, Cuda, Yolo version mismatch? 

### Environment

Sysinfo:
Python : Python 3.8.10
Ubuntu 20.04.4 LTS 64 bit

nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2020 NVIDIA Corporation
Built on Mon_Oct_12_20:09:46_PDT_2020
Cuda compilation tools, release 11.1, V11.1.105
Build cuda_11.1.TC455_06.29190527_0

Pip install -r requirements.txt:

Requirement already satisfied: matplotlib>=3.2.2 in /home/aris/.local/lib/python3.8/site-packages (from -r requirements.txt (line 4)) (3.5.1)
Requirement already satisfied: numpy>=1.18.5 in /home/aris/.local/lib/python3.8/site-packages (from -r requirements.txt (line 5)) (1.21.4)
Requirement already satisfied: opencv-python>=4.1.2 in /home/aris/.local/lib/python3.8/site-packages (from -r requirements.txt (line 6)) (4.5.3.56)
Requirement already satisfied: Pillow>=7.1.2 in /home/aris/.local/lib/python3.8/site-packages (from -r requirements.txt (line 7)) (8.4.0)
Requirement already satisfied: PyYAML>=5.3.1 in /usr/lib/python3/dist-packages (from -r requirements.txt (line 8)) (5.3.1)
Requirement already satisfied: requests>=2.23.0 in /home/aris/.local/lib/python3.8/site-packages (from -r requirements.txt (line 9)) (2.26.0)
Requirement already satisfied: scipy>=1.4.1 in /home/aris/.local/lib/python3.8/site-packages (from -r requirements.txt (line 10)) (1.8.0)
Requirement already satisfied: torch>=1.7.0 in /usr/lib/python3/dist-packages (from -r requirements.txt (line 11)) (1.10.1)
Requirement already satisfied: torchvision>=0.8.1 in /usr/lib/python3/dist-packages (from -r requirements.txt (line 12)) (0.12.0)
Requirement already satisfied: tqdm>=4.41.0 in /home/aris/.local/lib/python3.8/site-packages (from -r requirements.txt (line 13)) (4.62.3)
Requirement already satisfied: tensorboard>=2.4.1 in /usr/lib/python3/dist-packages (from -r requirements.txt (line 16)) (2.7.0)
Requirement already satisfied: pandas>=1.1.4 in /home/aris/.local/lib/python3.8/site-packages (from -r requirements.txt (line 20)) (1.4.1)
Requirement already satisfied: seaborn>=0.11.0 in /home/aris/.local/lib/python3.8/site-packages (from -r requirements.txt (line 21)) (0.11.2)
Requirement already satisfied: thop in /home/aris/.local/lib/python3.8/site-packages (from -r requirements.txt (line 37)) (0.0.31.post2005241907)
Requirement already satisfied: pyparsing>=2.2.1 in /home/aris/.local/lib/python3.8/site-packages (from matplotlib>=3.2.2->-r requirements.txt (line 4)) (2.4.7)
Requirement already satisfied: packaging>=20.0 in /home/aris/.local/lib/python3.8/site-packages (from matplotlib>=3.2.2->-r requirements.txt (line 4)) (21.3)
Requirement already satisfied: fonttools>=4.22.0 in /home/aris/.local/lib/python3.8/site-packages (from matplotlib>=3.2.2->-r requirements.txt (line 4)) (4.29.1)
Requirement already satisfied: python-dateutil>=2.7 in /home/aris/.local/lib/python3.8/site-packages (from matplotlib>=3.2.2->-r requirements.txt (line 4)) (2.8.2)
Requirement already satisfied: kiwisolver>=1.0.1 in /home/aris/.local/lib/python3.8/site-packages (from matplotlib>=3.2.2->-r requirements.txt (line 4)) (1.3.1)
Requirement already satisfied: cycler>=0.10 in /usr/lib/python3/dist-packages (from matplotlib>=3.2.2->-r requirements.txt (line 4)) (0.10.0)
Requirement already satisfied: idna<4,>=2.5; python_version >= "3" in /home/aris/.local/lib/python3.8/site-packages (from requests>=2.23.0->-r requirements.txt (line 9)) (2.10)
Requirement already satisfied: charset-normalizer~=2.0.0; python_version >= "3" in /home/aris/.local/lib/python3.8/site-packages (from requests>=2.23.0->-r requirements.txt (line 9)) (2.0.7)
Requirement already satisfied: certifi>=2017.4.17 in /home/aris/.local/lib/python3.8/site-packages (from requests>=2.23.0->-r requirements.txt (line 9)) (2021.5.30)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /home/aris/.local/lib/python3.8/site-packages (from requests>=2.23.0->-r requirements.txt (line 9)) (1.26.6)
Requirement already satisfied: pytz>=2020.1 in /home/aris/.local/lib/python3.8/site-packages (from pandas>=1.1.4->-r requirements.txt (line 20)) (2021.3)
Requirement already satisfied: six>=1.5 in /usr/lib/python3/dist-packages (from python-dateutil>=2.7->matplotlib>=3.2.2->-r requirements.txt (line 4)) (1.14.0)
 

### Minimal Reproducible Example

```
python path/to/detect.py --weights yolov5s.pt --source 0 
```

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@aris-t it appears you may have environment problems. Please ensure you meet all dependency requirements if you are attempting to run YOLOv5 locally.  If in doubt, create a new virtual [Python 3.9](https://www.python.org/) environment, clone the latest repo ([code changes daily](https://github.com/ultralytics/yolov5/commits/master)), and `pip install` [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) again from scratch.

💡 ProTip! Try one of our verified environments below if you are having trouble with your local environment.


## Requirements

[**Python>=3.7.0**](https://www.python.org/) with all [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) installed including [**PyTorch>=1.7**](https://pytorch.org/get-started/locally/). To get started:
```bash
git clone https://github.com/ultralytics/yolov5  # clone
cd yolov5
pip install -r requirements.txt  # install
```

[Models](https://github.com/ultralytics/yolov5/tree/master/models) and [datasets](https://github.com/ultralytics/yolov5/tree/master/data) download automatically from the latest YOLOv5 [release](https://github.com/ultralytics/yolov5/releases) when first requested.

## Environments

YOLOv5 may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Google Colab and Kaggle** notebooks with free GPU: <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>


## Status

![CI CPU testing](https://github.com/ultralytics/yolov5/workflows/CI%20CPU%20testing/badge.svg)

If this badge is green, all [YOLOv5 GitHub Actions](https://github.com/ultralytics/yolov5/actions) Continuous Integration (CI) tests are currently passing. CI tests verify correct operation of YOLOv5 training ([train.py](https://github.com/ultralytics/yolov5/blob/master/train.py)), validation ([val.py](https://github.com/ultralytics/yolov5/blob/master/val.py)), inference ([detect.py](https://github.com/ultralytics/yolov5/blob/master/detect.py)) and export ([export.py](https://github.com/ultralytics/yolov5/blob/master/export.py)) on MacOS, Windows, and Ubuntu every 24 hours and on every commit.
